### DISCUSSION TITLE
Welcome to postfixadmin discussions!

### DISCUSSION THREAD

**Discussion by DavidGoodwin** [postfixadmin/postfixadmin#403] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NicoloLazzaroni** [comment]
@DavidGoodwin, Idea for the next version of PostfixAdmin: Multiple databases to manage on one interface. You can choose if you want to add a domain on DB1 or DB2, that could make one interface manage multiple mail servers.

An example would be:

```
$dbh1 = mysql_connect($hostname, $username, $password); 
$dbh2 = mysql_connect($hostname, $username, $password, true); 

mysql_select_db('database1', $dbh1);
mysql_select_db('database2', $dbh2);

mysql_query('select * from tablename', $dbh1);
```

**Reply by DavidGoodwin** [reply]
Hi - can you elaborate as to why would you want to do this? 

Postfixadmin allows you to have administrators per domain - so couldn't everything just be in one database? :-)

**Reply by NicoloLazzaroni** [reply]
Hi,
Well, that may be used if a person uses multiple databases, for security reasons and to avoid general downtimes, so that if the main server is down, the second one will still work.

**Reply by DavidGoodwin** [reply]
Would you be better off using MySQL replication (or similar) if you want to improve availability?

**Reply by NicoloLazzaroni** [reply]
Hi,
That may be an option. But I still think that it may be a good security option to have different databases.

Example:
Customer 1,2,3 on Server 1, DB1.

Customer 4 alone on server 2, DB2.

Customer 5, 6 and 7 on DB3.

Also because replicator would send useless informations to the server so storage usage or send datas that won't be needed (eg. Customer 1 datas on Customer 4 server) and that may be a security problem.


So that an example configuration would be:

```
PFA on a strong WebHost with the WebMail.
Post Server 1 and DB on Server 1
Post Server 2 and DB on Server 2
```

Not many resources used, server are completely indipendent, if one is vulnerable it would not expose the other one's datas.

It is not essentials as feature, I'm just making a suggestion for the future. :D

Hope I explained everything and still congratulations for the project.

Best Regards

**Comment by realsaler** [comment]
I do Dropship everyday we use about 50 alias from getting tracking FedEx Ups I wish if there anything like
user registration form
add alias from user account
![alias](https://user-images.githubusercontent.com/59805585/141174874-3d7d551e-8693-4a2e-ba74-82b81457eed8.PNG)

Example mail.com
![alias](https://user-images.githubusercontent.com/59805585/141175711-96578b6b-1112-46d7-9784-628026dc7827.PNG)

**Reply by DavidGoodwin** [reply]
Postfixadmin allows an admin to create aliases/forwards for an account, but end users can't create them.

I suspect allowing an end user to create an arbitrary alias would be problematic for almost all organisations.

**Comment by CLechleitner42** [comment]
Hi!

I'm Christoph Lechleitner from iteg.at, a small IT company in Innsbruck, Austria.
We are using postfixadmin for some 10 years now for ourselfes and a few hosting customers. 

Almost 10 years ago, in 2013, I contributed to the `vacation.pl` script, allowing to include the original mail's subject in the subject of the automatic reply, using a hardcoded `$SUBJECT` pseudo variable. The commit now lives here: https://github.com/postfixadmin/postfixadmin/commit/2e3998c14c7ba88e9a953a1d44c435a2815ca724. I appreciate history was preserved when moving to Github ;-)

Then in 2018 there was some issue with the debian package, and we started providing a debian repository for some 3.1.x version.

A week ago we upgraded our mail server from Debian 10 Buster to Debian 11 Bullseye and found out that Debian seems to have dropped the `postfixadmin` package for bad, so I did it (the "fork" + debian repo) again, for 3.3.11 this time. More in "Show and tell" shortly-ish.
PS: I just found out `postfixadmin` is still (or back?) in Debian `testing` and `unstable`, I'll stick to our package for Debian 11 Bullseye (current `stable`) anyway.

I already love some of the enhancements in 3.3 (compared to 3.1), particularily the inclusion of time of day in the vacation dates.

Regards, Christoph


PPS: Sorry for the edits.
